appveyor.yml

```diff
@@ -56,11 +57,12 @@ for:
       - ps: .\install_windows.ps1
     test_script:
     - ps: |
-        git checkout ${DRIVER_LATEST_TAG}
+        git checkout $env:DRIVER_LATEST_TAG
 
         $env:CASSANDRA_VERSION=$env:CCM_VERSION
         $env:CCM_SSL_PATH="$env:CCM_PATH\ssl"
         $env:DOTNET_CLI_TELEMETRY_OPTOUT=1
+        $env:BuildAllTargets="True"
```

Comment:
Do we need build all targets? smoke testing the server against 1 target in windows should be good enough.

Reply:
Hmm true, this just "enables" the possibility of building all targets, we still build one only with the `-f` flag but it's unnecessary in any case, I'll remove it